test/repl.jl

```diff
@@ -870,6 +870,17 @@ end
     end end end
 end
 
+@testset "preview" begin
+    temp_pkg_dir() do project_path; cd_tempdir() do tmpdir; with_temp_env() do;
+        pkg"add Example"
+        pkg"preview 'rm Example'"
```

Comment:
Why the `'`s?

Reply:
The main reason is that it avoids any special parsing for `preview`. But now that I think about, an equally noninvasive fix would be to replace `preview` with a `--preview` meta option.

Reply:
So `pkg> preview add Example` doesn't work anymore? You need `pkg> preview 'add Example'`?

Reply:
If we really want `pkg> preview add Example` syntax though I'll bite the bullet and get it done.

Reply:
I've opted to go with both, in case the meta option syntax is a breaking change. The code would be cleaner with `preview` as just a meta option, but perhaps its too late for that?